lib/cylc/scheduler.py

```diff
@@ -251,15 +222,13 @@ def configure( self ):
         self.request_handler = request_handler( self.pyro )
         self.request_handler.start()
 
-        # LOAD TASK POOL ACCORDING TO STARTUP METHOD
+        self.suite_state = StateSummaryServer(
+            self.config, self.run_mode, str(self.initial_point))
```

Comment:
I had thought that the change from `str(self.pool.get_min_point())` to `str(self.initial_point)` might cause problems during a restart, but it turns out only 5.4.x gcylcs actually use the resulting `start time` property. They won't work for a whole host of other reasons with cylc 6 suites anyway. Can we remove this third argument and the `start time` global summary key?